barretenberg/cpp/pil/avm/constants_gen.pil

```diff
@@ -1,9 +1,9 @@
 // GENERATED FILE - DO NOT EDIT, RUN yarn remake-constants in circuits.js
 namespace constants(256);
-    pol MAX_NEW_NOTE_HASHES_PER_CALL = 16;
-    pol MAX_NEW_NULLIFIERS_PER_CALL = 16;
+    pol MAX_NOTE_HASHES_PER_CALL = 16;
+    pol MAX_NULLIFIERS_PER_CALL = 16;
     pol MAX_PUBLIC_CALL_STACK_LENGTH_PER_CALL = 16;
-    pol MAX_NEW_L2_TO_L1_MSGS_PER_CALL = 2;
+    pol MAX_L2_TO_L1_MSGS_PER_CALL = 2;
     pol MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_CALL = 32;
```

Comment:
You'll need to run `./compile_avm.sh` to regenerate C++ code after this change. We'll try to improve this tooling.

Reply:
The script seems to be broken:
<img width="572" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/a1550bce-4007-47a9-9d6b-21b60ee5b219">

and when I tried fixing that by changing the content to:

```
#!/bin/bash

# Compile
../../../bb-pilcom/target/release/bb_pil pil/avm/main.pil --name Avm
```

I got:

#!/bin/bash

# Compile
../../../bb-pilcom/target/release/bb_pil pil/avm/main.pil --name Avm

Reply:
Make sure to run it from `barretenberg/cpp`. Also note that it'll change a lot of the files, but they'll be auto-formatted when you commit them so the diff won't as big as it seems.

Reply:
Managed to run it and after it got formatted it didn't result in any changes https://github.com/AztecProtocol/aztec-packages/pull/7273/commits/b96023a7d6dfa52511dd6a80d9468bf2b3f89c51

Reply:
@benesjan It is normal because pil constants are completely inlined (replaced by literal value) in the cpp generated relations.